src/panels/lovelace/components/hui-warning.ts

```diff
@@ -18,19 +19,7 @@ export const createEntityNotFoundWarning = (
 @customElement("hui-warning")
```

Comment:
This element now is the same as `ha-alert` with the `warning` type? Not better to just remove it?

Reply:
Yes, but that will break any custom card that uses it, which is why I did not do that.